telco-hub/configuration/reference-crs/optional/logging/clusterLogSubscription.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: cluster-logging
   namespace: openshift-logging
 spec:
-  channel: "stable-6.2"
+  channel: "stable-6.3"
```

Comment:
I've bumped the channel version for `cluster-logging` operator given that the default one is now `stable-6.3` in production catalogs

```
NAME             DISPLAY NAME  DEFAULT CHANNEL
cluster-logging                stable-6.3

PACKAGE          CHANNEL     HEAD
cluster-logging  stable-6.2  cluster-logging.v6.2.3
cluster-logging  stable-6.3  cluster-logging.v6.3.0
```

Reply:
As agreed during offline discussion, we will remain in stable-6.2 due to EUS-EUS supportability